src/index.ts

```diff
@@ -196,7 +202,7 @@ const refreshToken = async (requestRefresh: TokenRefreshRequest): Promise<Token>
   }
 }
 
-export type TokenRefreshRequest = (refreshToken: string) => Promise<Token>
+export type TokenRefreshRequest = (refreshToken: string) => Promise<Token | IAuthTokens>
```

Comment:
`requestRefresh` returns `Promise<IAuthTokens | string>`, but here it's `Promise<Token | IAuthTokens>`.

Is `Token` just a string?

Reply:
yes (this hasn't changed this)

Reply:
I've updated the code so that tokens are always types as `Token` and not `string`